share/nisar/defaults/static.yaml

```
@@ -0,0 +1,436 @@
+          # Must be >= 0.
+          # Defaults to 0.
+          az_margin: 0.0
+          # [OPTIONAL] Additional slant range margin, in meters, to add to the near and
+          # far edges of the radar grid used to form static layers.
+          # Must be >= 0.
+          # Defaults to 0.
+          rg_margin: 0.0
+
+      dem:
+        # [OPTIONAL] Interpolation method used for sampling the input DEM raster.
+        # Must be one of:
+        #   - 'nearest': nearest neighbor interpolation
+        #   - 'bilinear': bilinear interpolation
+        #   - 'bicubic': bicubic interpolation
+        #   - 'biquintic': biquintic spline interpolation
+        #   - 'sinc': interpolation using a separable 9x9 sinc kernel
+        # Defaults to 'biquintic'.
+        interp_method: biquintic
+
+      water_mask:
+        # [OPTIONAL] Algorithm to use for resampling the input water mask to the desired
+        # output geographic coordinate grid.
+        # Must be one of:
+        #   - 'near': nearest neighbor resampling
+        #   - 'mode': mode resampling (selects the value which appears most often among
+        #     sampled points)
+        # Defaults to 'near'.
+        resample_algorithm: near
+
+      topo:
+        # [OPTIONAL] Number of range lines per block used by the topo algorithm to
+        # compute the layover/shadow mask.
+        # Must be >= 1.
+        # Defaults to 1024.
+        lines_per_block: 1024
+
+      geocode:
+        # [OPTIONAL] Algorithm used for geocoding.
+        # Must be one of:
+        #   - 'interp': interpolation
+        #   - 'area_projection': pixel area-based adaptive multilooking algorithm
+        # Defaults to 'area_projection'.
+        output_mode: area_projection
+        # [OPTIONAL] Interpolation method used for geocoding.
+        # Only relevant when `output_mode`='interp'. Ignored otherwise.
+        # Must be one of:
+        #   - 'nearest': nearest neighbor interpolation
+        #   - 'bilinear': bilinear interpolation
+        #   - 'bicubic': bicubic interpolation
+        #   - 'biquintic': biquintic spline interpolation
+        #   - 'sinc': interpolation using a separable 9x9 sinc kernel
+        # Defaults to 'biquintic'.
+        interp_method: biquintic
+        # [OPTIONAL] Block processing mode to use for geocoding.
+        # The choice of block processing mode may affect the peak memory utilization and
+        # runtime of the process.
+        # Must be one of:
+        #   - 'single_block': process the full scene as a single block
+        #   - 'geogrid': partition the geographic image grid into sub-blocks; process
+        #     the full radar image grid as a single block
+        #   - 'geogrid_and_radargrid': partition the geographic and radar image grids
+        #     into sub-blocks
+        #   - 'auto': the block processing mode is chosen by the software
+        # Defaults to 'geogrid_and_radargrid'.
+        memory_mode: geogrid_and_radargrid
```

```suggestion
        memory_mode: auto
```
Let's keep auto here and tell the SDS to use `geogrid_and_radargrid` to generate products with fine spacing.